LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -162,6 +202,66 @@ module LibraBFT.ImplShared.Consensus.Types where
   α-EC-RM : RoundManager → EpochConfig
   α-EC-RM rm = α-EC ((_rmEC rm) , (_rmEC-correct rm))
 
+
+  record GoodLens {A} (l : Lens RoundManagerEC A) : Set where
```

Comment:
How about something like`RMLens` for the name?

Reply:
Sounds good, thanks.  Done.